### DISCUSSION TITLE
TBS Lucid FC - Serial Rx Not Responding

### DISCUSSION THREAD

**Discussion by NoSadFaces** [iNavFlight/inav#10321] [Support]
Hello, I'm a beginner to iNav and have been having trouble getting iNav to communicate with my Crossfire Nano receiver.
The current build works in Betaflight, but when I flash iNav to the TBS Fusion FC, I can't get the serial Rx  to recognize the receivers inputs. 

TBS Lucid FC
Crossfire Nano Receiver
Using iNav 7.1.2

Thanks for any help.

**Comment by mmosca** [comment]
Can you share a screenshot of your ports tab and receiver tab?

You can also try a nightly build of INAV 8.0.0.

**Reply by NoSadFaces** [reply]
I updated to the most recent 8.0.0 release and still have the same issue.

![ports](https://github.com/user-attachments/assets/1d9a7714-7270-4a34-b854-6db200911c2f)
![receiver](https://github.com/user-attachments/assets/3d4e73d6-566d-45a3-9104-0d2d9150ef0f)

My setup follows the same UART layout as the manual. Also tested running Betaflight again, and the UART works properly.
https://www.team-blacksheep.com/media/files/tbs-lucid-manual.pdf

**Comment by mmosca** [comment]
@bkleiner can you have a look at this one? I tested with a gps and it worked, but looks like higher baud rates may have issues. (Tested with elrs rx on uart 5 and 7 and it did not work.

**Reply by bkleiner** [reply]
Will do. I am off on holiday till mid next week, but I will look into it after.
Edit: Now that I think about it pretty sure I tried it with a nano rx. 

@NoSadFaces could you please check which crsf version you are running?

**Reply by NoSadFaces** [reply]
Nano RX Firmware v6.36 on hardware v1.33.

**Reply by mmosca** [reply]
I will recheck later today. Turns out my elrs radio was setup for mavlinkrc, not crsf. But I did see traffic on the uart using a scope.

**Reply by bkleiner** [reply]
just tried it again on inav 7.1.2 with a nano rx v1.01 on v6.36 and it indeed works as expected. continuing to investigate. 
update: happymodel elrs receiver also seems to work fine.
@mmosca can you please see if the issue on your end was indeed related to your receiver settings.
@NoSadFaces which target do you flash on BF? there are two revisions of this board.
